### PR TITLE
[RAPTOR-8213] update pypmml environment to Python 3.9

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

**Comment by yakov-g** [comment]
jenkins run black_check

File: public_dropin_environments/python3_pmml/dr_requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-pyarrow==0.14.1
-datarobot-drum==1.9.4
+pyarrow>=0.14.1,<=3.0.0
```
Comment (akshoop) [inline public_dropin_environments/python3_pmml/dr_requirements.txt:1]: Do we want to have such a high upper constraint for `pyarrow`? I see `pyarrow==3.0.0` was released on Jan 2021 ([LINK](https://pypi.org/project/pyarrow/3.0.0/))

**Review by akshoop — APPROVED** [review]
LGTM from Core Modeling

File: public_dropin_environments/python3_pmml/dr_requirements.txt (hunk 1 above)
Comment (yakov-g) [reply public_dropin_environments/python3_pmml/dr_requirements.txt:1]: This is the first version supporting py3.9. Should we use a version  that supports 3.9, or earlier can be used?

**Comment by yakov-g** [comment]
jenkins run black_check

File: public_dropin_environments/python3_pmml/dr_requirements.txt (hunk 1 above)
Comment (akshoop) [reply public_dropin_environments/python3_pmml/dr_requirements.txt:1]: Oh my bad forgot that you're upgrading to a version that supports python3.9. Then yes, `pyarrow==3.0.0` is the first latest version of pyarrow that supports python3.9 so let's go with that. Thanks for clarifying!

**Comment by yakov-g** [comment]
jarvis run pipelineController:all-expected-tasks

**Review by meltzg — APPROVED** [review]

**Comment by yakov-g** [comment]
jarvis run cmrunner_integration

**Comment by yakov-g** [comment]
jenkins run cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Pull request merged**